session-keys/app/src/index.css

```
@@ -0,0 +1,233 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
```

Stylelint (17.11.1)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap'" (import-notation)

(import-notation)